deploy/Jenkinsfile

```diff
@@ -0,0 +1,227 @@
+pipeline {
+    agent {
+        label 'bb5'
+    }
+    stages {
+
+        stage('build AoS'){
+            steps {
+                sh 'sh deploy/install.sh AoS'
+            }
+        }
+
+        stage('build SoA'){
+            steps {
+                sh 'sh deploy/install.sh SoA'
+            }
+        }
+
+        stage('ringtest'){
+            parallel{
+                stage('AoS'){
+                    steps{
+                        sh 'sh deploy/ringtest.sh AoS'
+                    }
+                }
+                stage('SoA'){
+                    steps{
+                        sh 'sh deploy/ringtest.sh SoA'
+                    }
+                }
+            }
+        }
+
+        stage('checkout testcorenrn'){
+            steps{
+                sh 'cd $WORKSPACE'
+                sh 'rm -rf testcorenrn'
+                checkout([
+                    $class: 'GitSCM',
+                    branches: [[name: 'master']],
+                    doGenerateSubmoduleConfigurations: false,
+                    extensions: [[$class: 'RelativeTargetDirectory', relativeTargetDir: 'testcorenrn']],
+                    submoduleCfg: [],
+                    userRemoteConfigs: [[url: 'https://github.com/pramodk/testcorenrn.git']]
+                ])
+            }
+        }
+
+        stage('nrnivmodl'){
+            parallel{
+                stage('nrnivmodl'){
+                    steps{
+                        sh 'sh deploy/nrnivmodl.sh'
```

Comment:
I have the impression that these scripts work in the same directory, even in running in parallel. Did I miss smtg?

Reply:
Yes, they all work in the same directory in parallel but do different tasks.